jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/AttachHandler.java

```diff
@@ -390,7 +390,7 @@ protected boolean terminate(boolean wakeHandler) {
 			if (LOGGING_DISABLED != loggingStatus) {
 				IPC.logMessage("AttachHandler terminate removing directory : ", TargetDirectory.getTargetDirectoryPath(getVmId())); //$NON-NLS-1$
 			}
-			TargetDirectory.deleteMyDirectory(false);
+			TargetDirectory.deleteMyDirectory(false, “”);
```

Comment:
This appears a full width string, and appeared as `??????` after JPP processing which is required to generate Java level specific pConfig files.
Please change `“”` to `""`.

Reply:
OMG, sorry for this inconvenience.

Reply:
```suggestion
			TargetDirectory.deleteMyDirectory(false, "");
```

Reply:
No worries.